client/state/stats/lists/utils.js

```diff
@@ -397,6 +410,93 @@ export const normalizers = {
 		} );
 	},
 
+	/**
+	 * Returns a normalized payload from `/sites/{ site }/stats/archives`
+	 * @param   {Object} data    Stats data
+	 * @param   {Object} query   Stats query
+	 * @returns {Array}          Normalized stats data
+	 */
+	statsArchives: ( data, query ) => {
+		if ( ! data || ! query.period || ! query.date ) {
+			return [];
+		}
+
+		const { startOf } = rangeOfPeriod( query.period, query.date );
+		const dataPath = query.summarize ? [ 'summary' ] : [ 'days', startOf ];
+		const archivesData = get( data, dataPath, [] );
+
+		const archives = Object.keys( archivesData ).reduce( ( accumulatedArchives, archiveKey ) => {
+			const archiveItems = archivesData[ archiveKey ];
+
+			// Taxonomy items are grouped by taxonomy term.
+			if ( 'tax' === archiveKey ) {
+				let totalTaxViews = 0;
+
+				const taxItems = Object.keys( archiveItems ).map( ( taxKey ) => {
+					const taxItem = archiveItems[ taxKey ];
+					const hasSubItems = Array.isArray( taxItem ) && taxItem.length > 0;
+					let itemViews = 0;
+
+					if ( hasSubItems ) {
+						const children = taxItem.map( ( item ) => {
+							itemViews += item.views;
+							totalTaxViews += item.views;
+
+							return {
+								label: item.value,
+								value: item.views,
+								link: item.href,
+							};
+						} );
+
+						return {
+							label: taxKey,
+							value: itemViews,
+							children,
+						};
+					}
+
+					return {
+						label: taxKey,
+						value: itemViews,
+					};
+				} );
+
+				accumulatedArchives.push( {
+					label: getArchiveKeyLabel[ archiveKey ],
+					value: totalTaxViews,
+					children: taxItems,
+				} );
+			} else {
+				const hasItems = Array.isArray( archiveItems ) && archiveItems.length > 0;
+
+				if ( hasItems ) {
+					let totalViews = 0;
+
+					const children = archiveItems.map( ( item ) => {
+						totalViews += item.views;
+
+						return {
+							label: [ 'home', 'search' ].includes( archiveKey ) ? item.href : item.value,
+							value: item.views,
+							link: item.href,
+						};
+					} );
+
+					accumulatedArchives.push( {
+						label: getArchiveKeyLabel[ archiveKey ],
```

Comment:
```suggestion
						label: getArchiveKeyLabel()[ archiveKey ],
```

`getArchiveKeyLabel` is a function at this point, I believe, correct?

Reply:
Thanks for the review. Fixed in [96b683d](https://github.com/Automattic/wp-calypso/pull/103945/commits/96b683d486a2c19abd53af1a694e8514b418743b).